tests/commands/ddtrace_run_sitecustomize.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function
+
+import sys
+from ddtrace import tracer
+from nose.tools import ok_
+
+
+if __name__ == '__main__':
+    # detect if `-S` is used
```

Comment:
@SeanOC if `-S` is used, `sitecustomize` must not be in cached modules. This makes the regression test even better, because we know:
* a `sitecustomize.py` has been auto-loaded
* it's not ours because of `CORRECT_IMPORT`